DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: Rnw2Rmd
 Title: Initial conversion from Rnw to Rmd
-Version: 0.99.2
+Version: 0.100.0
```

Comment:
I don't think this version number makes sense from a Bioconductor perspective? Submissions are 0.99.x, versions always have to go up. Going up from 100 doesn't lead to 99.

Reply:
Corrected to 0.99.3, thanks @mtmorgan